frontend/gist-frontend/src/gist-login/LoginPageComponent.tsx

```diff
@@ -8,13 +8,21 @@ import { Box, Grommet, Tab, Tabs, Heading } from "grommet";
 const theme = {
```

Comment:
eventually move theme into it's own file

Reply:
done and done